packages/bot-web-ui/src/components/load-modal/google-drive/__tests__/google-drive.spec.tsx

```diff
@@ -53,7 +63,12 @@ describe('GoogleDrive', () => {
         );
     });
 
-    it('should render GoogleDrive component with svg element that has height and width equal 128, and should have text Google Drive, when is_authorised = false on desktop version', () => {
+    it('should render', () => {
+        const { container } = render(<GoogleDrive />, { wrapper });
+        expect(container).toBeDefined();
+    });
+
+    it('should render GoogleDrive component with svg element that has height and width equal 128', () => {
```

Comment:
can we remove GoogleDrive component ? because we don't have on the first test case ?